panel/pane/base.py

```diff
@@ -545,7 +545,7 @@ class ReplacementPane(PaneBase):
 
     _pane = param.ClassSelector(class_=Viewable)
 
-    _ignored_refs: ClassVar[Tuple[str]] = ['object']
+    _ignored_refs: ClassVar[Tuple[str,...]] = ('object',)
```

Comment:
Please note the types and values of `_ignore_refs` were incorrect. I believe I've fixed them correctly in the places where needed @philippjfr